### ISSUE TITLE
PolyStrips: Unhandled Exception Caught: Exception caught

### ISSUE THREAD

**Issue by Galilla** [CGCookie/retopoflow#1032]
**PLEASE READ THE FOLLOWING AND USE THE TEMPLATE TO POST YOUR ISSUE**

*Questions not directly related to the RetopoFlow project are considered general and should be posted elsewhere (e.g. git, blender, python, etc).*

**Before raising the issue, please check the following first:**

- [ ] Read through the README.md file in the branch used to assure the process taken is correct
- [ ] Check the existing [issues](https://github.com/CGCookie/retopoflow/issues) and [pull requests](https://github.com/CGCookie/retopoflow/pulls) to make sure the issue has not already been reported and/or fixed.
- [ ] You can replicate the bug and will provide as much info as possible with blend files, screenshots, logfiles, as well as show the expected result and actual result.

Please prefix your issue name with one of the following: **[BUG]** **[PROPOSAL]** **[QUESTION]**

Issue Template
-----------------------------------

**Issue:**

{*enter your issue here*}

{*add any screenshots of the issue here*}

**How to Reproduce:**

- {*add your list replication steps*}

{*attach any files needed to replicate the issue*}

Automatically Generated Info
-----------------------------------

```
Environment:

- RetopoFlow: 3.2.0
- CG Cookie built
- Blender: 2.92.0 master 2021-02-24
- Platform: Windows, 10, 10.0.19041, AMD64, Intel64 Family 6 Model 158 Stepping 10, GenuineIntel
- GPU: NVIDIA Corporation, GeForce RTX 2080/PCIe/SSE2, 4.5.0 NVIDIA 441.06, 4.50 NVIDIA
- Timestamp: 2021-07-25 13:35:16.246470
- Undo: manipulate bezier, stroke, select, select, tweak move, tweak move, tweak move, select, extrude cycle, select

Error Hash: a1f11c0c39bfe8f951d1994658693fd6

Trace:

- EXCEPTION (<class 'TypeError'>): 'NoneType' object is not iterable
-     C:\Users\Gail\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\RetopoFlow\retopoflow\rfmesh\rfmesh_wrapper.py
- 000 0162:co() if any(math.isnan(v) for v in co): return
-     C:\Users\Gail\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\RetopoFlow\retopoflow\rftool_polystrips\polystrips_utils.py
- 001 0330:update() bmv0.co = v0
-     C:\Users\Gail\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\RetopoFlow\retopoflow\rftool_polystrips\polystrips.py
- 002 0313:movehandle() strip.update(self.rfcontext.nearest_sources_Point, self.rfcontext.raycast_sources_Point, self.rfcontext.update_face_normal)
-     .../common\fsm.py
- 003 0058:run() return fn(*args, **kwargs)
```

**Comment by vxlcoder** [comment]
thanks for the report, @Galilla.

i believe this issue should be fixed in commit 810d148, but i'm not sure how to reproduce this issue in order to fully test the fix.

can you provide the steps you used to run into this issue?

**Comment by Galilla** [comment]
Hi Jon,

No, sorry, I did fix it but can't remember now.  I had two different errors
that seemed to be new but I just purchased the program yesterday and am not
comfortable with it yet.

Galilla

**Comment by vxlcoder** [comment]
ok.  thanks for the info!

i believe the commit above _should_ fix the issue, but i wanted to try reproducing it before declaring it solved :)

**Comment by Galilla** [comment]
I keep getting stray vertices creating faces at the origin.  I can't see
these until I export as obj and import it again.
Could this be a result of accidentally dragging a line off the model?

By the way, I tried to submit both errors but I didn't think they went
through because the website said it failed.
The first time I tried to submit I had to go through forgot password, etc,
and figured it was that.
The second time my account was good but it still said it failed to submit.

Thanks again for your help
Galilla
